src/app/shared/components/image-form-control/image-form-control.component.ts

```diff
@@ -81,13 +85,13 @@ export class ImageFormControlComponent implements OnInit, ImageFormControlCompon
     });
   }
 
-  onChange = (array: File[]): void => { };
-  onTouched = (): void => { };
+  onChange: FilesToVoid;
```

Comment:
please move these  fields above ngoninit method, since these are instance fields, not methods.